### PR TITLE
Migrate to GH Actions

### LINKED ISSUE #358: global: missing pypi-publish workflow
The pypi-publish.yml workflow is missing.

### PR DESCRIPTION
**Pull request by avivace**
Closes #358 and #360

### CODE REVIEW

**Comment by mvidalgarcia** [comment]
There are still many things missing for the migration of this repo. I see several references to travis, including `.travis.yml`. Should we merge this now or wait until the `ga-migration` script + `docker-services-cli` is fully ready?

**Comment by avivace** [comment]
> There are still many things missing for the migration of this repo. I see several references to travis, including `.travis.yml`. Should we merge this now or wait until the `ga-migration` script + `docker-services-cli` is fully ready?

I guess the other thing you mentioned can go in another PR? Then, running again the ga-migration script will just update the rest (merging shouldn't generate conflicts)

**Comment by diegodelemos** [comment]
@avivace Can you rerun the `ga-migration` script, fix changes locally, amend the commit, and force push. So we don't have to revisit this repo.

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Publish
```
Comment (lnielsen) [inline .github/workflows/pypi-release.yml:1]: Minor: This file should not be here - only the pypi-publish.yml is needed.

**Pull request merged**